### PR TITLE
fix bug #2128  "TypeError: not enough arguments for format string"

### PR DESCRIPTION
**Pull request by jinmenglei**
fix bug #2128 "TypeError: not enough arguments for format string" in DeserializationErroralizationError

### CODE REVIEW

File: clients/rospy/src/rospy/impl/tcpros_base.py [hunk 1]
```diff
@@ -817,8 +817,8 @@ def receive_loop(self, msgs_callback):
                 except DeserializationError as e:
                     #TODO: how should we handle reconnect in this case?
                     
-                    logerr("[%s] error deserializing incoming request: %s"%self.name, str(e))
-                    rospyerr("[%s] error deserializing incoming request: %s"%self.name, traceback.format_exc())
+                    logerr("[%s] error deserializing incoming request: %s"% (self.name, str(e)))
+                    rospyerr("[%s] error deserializing incoming request: %s"% (self.name, traceback.format_exc()))
```
Comment (fujitatomoya) [inline clients/rospy/src/rospy/impl/tcpros_base.py]: nitpick:
```suggestion
                    logerr("[%s] error deserializing incoming request: %s"%(self.name, str(e)))
                    rospyerr("[%s] error deserializing incoming request: %s"%(self.name, traceback.format_exc()))
```

**Review by fujitatomoya — COMMENTED** [review]
lgtm

**Review by fujitatomoya — APPROVED** [review]

**Comment by jinmenglei** [comment]
@fujitatomoya How long will it take to merge? I want to apply for another pr

**Comment by fujitatomoya** [comment]
@jinmenglei i am not sure about that, since i am not a maintainer.

CC: @sloretz @jacobperron friendly ping.

**Comment by jinmenglei** [comment]
@fujitatomoya thank you

**Review by jacobperron — APPROVED** [review]
Thanks for the fix!

**Pull request merged**